tests/test_examples.py

```diff
@@ -245,6 +246,10 @@ def test_gradient_accumulation(self):
         testargs = ["examples/by_feature/gradient_accumulation.py"]
         run_command(self.launch_args + testargs)
 
+    def test_gradient_accumulation_for_autoregressive_models(self):
+        testargs = ["examples/by_feature/gradient_accumulation_for_autoregressive_models.py"]
+        run_command(self.launch_args + testargs)
```

Comment:
Just a nit: this doesn't use gradient accumulation here since it uses the default of `1`